process/process.go

```diff
@@ -43,6 +43,10 @@ var signalMap = map[string]Signal{
 	"SIGTERM": SIGTERM,
 }
 
+var ErrNotWaitStatus = errors.New(
+	"unimplemented for systems where exec.ExitError.Sys() is not syscall.WaitStatus",
```

Comment:
This error message is not helpful to users, so should not really be exposed except under debug. However, it is more or less what was there before.